internal/service/project/data_source_projects.go

```diff
@@ -163,6 +163,11 @@ func (d *ProjectsDS) Schema(ctx context.Context, req datasource.SchemaRequest, r
 								},
 							},
 						},
+						"tags": schema.MapAttribute{
+							ElementType: types.StringType,
+							Optional:    true,
```

Comment:
```suggestion
```

Reply:
fixed in 35d729e4e186219cab55b274cb1af334b82e1645